app/models/group.rb

```diff
@@ -2,6 +2,7 @@ class Group < ApplicationRecord
   has_many :permissions
   has_many :users, through: :user_groups
   has_many :user_groups, inverse_of: :group
+  has_many :segments
```

Comment:
esto debe ser un segment. Siempre o casi siempre la llave foranea, en este caso el `:segment_id` esta en la tabla `groups`. Por lo tanto el modelo Group debe llevar un `belongs_to` y como es belongs to debe ser en `singular`. Igualmente la tienes que declarar del lado del modelo `Segment` donde va un `has_many`, En la tarjeta:

```
Agregar la relacion:

Un Segment Has many Groups 
```